iothub/device/src/ClientOptions.cs

```diff
@@ -8,6 +8,7 @@ namespace Microsoft.Azure.Devices.Client
 {
     /// <summary>
     /// Options that allow configuration of the device or module client instance during initialization.
+    /// These options cannot be updated once the client has been initialized.
```

Comment:
cannot be or should not be updated? Do you intend to say "Updating these options after the client has been initialized will not change the behavior of the client?"

Reply:
I like this - Updating these options after the client has been initialized will not change the behavior of the client